### ISSUE TITLE
Metrics are not changing

### ISSUE THREAD

**Issue by joeyearsley** [fchollet/keras#5033]
When running a model with more metrics such as recall and fbeta_score the logged results are exactly the same:

      59596/59596 [==============================] - 3173s - loss: 0.5636 - acc: 0.6712 - fbeta_score: 0.6712 - recall: 0.6712 - val_loss: 0.5500 - val_acc: 0.6742 - val_fbeta_score: 0.6742 - val_recall: 0.6742

Surely these should be different? 

- [x] Check that you are up-to-date with the master branch of Keras. You can update with:
pip install git+git://github.com/fchollet/keras.git --upgrade --no-deps

- [x] If running on TensorFlow, check that you are up-to-date with the latest version. The installation instructions can be found [here](https://www.tensorflow.org/get_started/os_setup).

- [ ] If running on Theano, check that you are up-to-date with the master branch of Theano. You can update with:
pip install git+git://github.com/Theano/Theano.git --upgrade --no-deps

- [ ] Provide a link to a GitHub Gist of a Python script that can reproduce your issue (or just copy the script here if it is short).

**Comment by 162** [comment]
I have the simlar problem. After looking through outputs of NN I found that it does only constant predictions. In other words output doesn`t depend on input in any way even though that constant output vector may slightly change from epoch to epoch.

**Comment by joggino** [comment]
Hi Anton,
I'm unsure if we are on about the same thing, my accuracy and loss change per epoch (and in the right direction), just the metrics such as **fbeta_score** are the same as **accuracy** every epoch.

    X: 59596/59596 [==============================] - 5212s - loss: 0.3173 - acc: 0.8588 - fbeta_score: 0.8588 - recall: 0.8588 - val_loss: 0.9224 - val_acc: 0.7520 - val_fbeta_score: 0.7520 - val_recall: 0.7520
    Y: 59596/59596 [==============================] - 5211s - loss: 0.2674 - acc: 0.8824 - fbeta_score: 0.8824 - recall: 0.8824 - val_loss: 0.5197 - val_acc: 0.8169 - val_fbeta_score: 0.8169 - val_recall: 0.8169
